src/__tests__/saveDocs.test.ts

```diff
@@ -213,7 +213,7 @@ it('should be similar batch vs one', async () => {
 
   const row = { id: '', key: 'preact', value: { rev: 'a' }, doc: preact };
```

Comment:
is this variable still used?

Reply:
yes the line just after in saveDocs.
saveDocs will be removed after I deploy